compiler/src/iree/compiler/Codegen/Common/PolynomialApproximationPass.cpp

```diff
@@ -35,6 +35,7 @@ class PolynomialApproximationPass
       mathPatterns.add<math::ErfPolynomialApproximation>(&getContext());
     } else {
       populateMathPolynomialApproximationPatterns(mathPatterns);
+      populateExpandRoundEvenPattern(mathPatterns);
```

Comment:
These arent exact expansions? Thats surprising.

Reply:
Perhaps we need a different pass here as this one is just a plain expansion but you had asked to guard it, but perhaps we spoke past each other.